src/components/SQForm/SQFormMaskedTextField.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MaskedInput from 'react-text-mask';
+import SQFormTextField from './SQFormTextField';
+
+function TextFieldMask({inputRef, mask, ...rest}) {
+  return (
+    <MaskedInput
+      {...rest}
```

Comment:
This looks awesome, I actually need something just like this =)
Could we maybe have `placeholderChar` and `showMask` be props with defaults that way they are able to be specified with inputProps and overridden if needed?

Reply:
@gdoohen1 Could you clarify what `placeholderChar` and `showMask` would be used for.
Is this something that occurs in FlexForm you're needing? If so, can you screenshot or record a Loom to show the intent of these 2 props?

Reply:
@SeanGroff I don't have a use-case at the moment to utilize these, I was more just asking from the perspective of erroring on the side of exposing something rather preventing the user from accessing it - but if we have strong UX standards of never showing masking and using the same placeholder than probably the extra level of safety in locking these up is the better option.

Reply:
Right on.
We try to keep the API small for ease of use then expand the props API as needs arise.
If the need arises for either of these features we'll enhance this component at that time.

Reply:
Alright that sounds good to me!